### ISSUE TITLE
Database conventions

### ISSUE THREAD

**Issue by sbordeyne** [cyian-1756/nsfw_game_updater#11]
We need some kind of conventions. 

- Any game that is unavailable for a given platform must have its link set to "-"
- Games that are paid, for now, need the value "paid" as their public build field value
- If a game needs to be played in the browser, add "open:" before the full link, on all supported platforms (usually linux, mac and windows, because flash supports those three platforms, but not android)
- All versionning needs to be in the format x.y.z.t, if the game uses dates, transform the date to iso format (i.e 2018-01-23) and replace the '-' by '.'. Only digits and dots are allowed.

**Comment by cyian-1756** [comment]
> If a game needs to be played in the browser, add "open:" before the full link, on all supported platforms (usually linux, mac and windows, because flash supports those three platforms, but not android)

I disagree here, AFAIK flash games can be played locally on windows natively using the adobe stand alone flash debugger and the debugger can run under mac and linux using WINE

Besides that I agree

**Comment by sbordeyne** [comment]
> I disagree here, AFAIK flash games can be played locally on windows natively using the adobe stand alone flash debugger and the debugger can run under mac and linux using WINE

Well, all the flash games in the database have a link to the swf file as well as to the compiled exe file for windows. And you can install flash on any browser IIRC, and open the swf inside the browser. no need for the debugger.e

**Comment by cyian-1756** [comment]
>  And you can install flash on any browser IIRC, and open the swf inside the browser. no need for the debugger.e

True. I say add a config option for opening the swf in the browser or downloading it locally
